CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
+You must do the following.
 
-  To migrate: In the unlikely event that your app relies on the spacing of the tab headings being a certain height on mobile and with JS disabled, you should make the necessary adjustments in your code.
+1. Update file paths.
+2. Update and add `data-module` attributes.
+3. Update CSS class names.
 
-  ([PR #1330](https://github.com/alphagov/govuk-frontend/pull/1330))
+##### Update file paths
 
-- Removes `govuk-grid-row` mixin
+You must add `govuk/` to your import file paths.
 
-  https://github.com/alphagov/govuk-frontend/pull/1090 copied govuk-grid-row
-  mixin to create a new concrete `.govuk-grid-row` class and marked the mixin as deprecated.
+If you're importing `node_modules/govuk-frontend/all.js` in your project, change this file path to `node_modules/govuk-frontend/govuk/all.js`.
 
-  To migrate you'll need to remove any references to `govuk-grid-row` mixin and use `.govuk-grid-row` class instead.
+If you’re importing a specific path, add `govuk/` after 'govuk-frontend/`. For example, if you're importing the button component:
 
-  ([PR #1343](https://github.com/alphagov/govuk-frontend/pull/1343))
+```nunjucks
+import Button from 'govuk-frontend/govuk/components/button/button'
+```
 
-- Remove grid-width mixin
+##### Update and add data-module attributes
 
-  https://github.com/alphagov/govuk-frontend/pull/1090 deprecated this mixin
-  and left it in as an alias to govuk-grid-width.
+You do not need to do anything if you're using Nunjucks macros and the [`initAll`](https://github.com/alphagov/govuk-frontend/blob/268b1e1b962d4bd293c5681868a2119f8b4ece60/docs/installation/installing-with-npm.md#option-1-include-javascript) function.
 
-  To migrate you'll need to replace any reference to the `grid-width` mixin with `govuk-grid-width`
+If you are not using the Nunjucks macros, add a `govuk-` prefix to `data-module` attribute values. For example:
 
-  ([PR #1342](https://github.com/alphagov/govuk-frontend/pull/1342))
+```html
+<div class="govuk-accordion" data-module="govuk-accordion">
+...
+</div>
+```
 
-- Remove the deprecated `$class` parameter from the `@govuk-grid-column` mixin
+The [button](https://design-system.service.gov.uk/components/button/) and [details](https://design-system.service.gov.uk/components/details/) components now also use the `data-module` attribute for initialisation. If you are not using Nunjucks macros, add:
 
-  To migrate, you'll need to update any places in your code where you use the
-  `@govuk-grid-column` mixin.
+- `data-module="govuk-button"` to each `<button>` HTML tag
+- `data-module="govuk-details"` to each `<details>` HTML tag
 
-  If you are currently passing a class name (`$class: 'foo'`) you should remove
-  the `$class` argument and wrap your call to the mixin in the class name
-  instead:
+If you're using your own JavaScript code to initialise components, add a `govuk-` prefix to any selectors that find components.
```

Comment:
I guess they only need to do this if they've been hooking onto the `data-module` to initialise. If they hook onto the CSS class that hasn't changed.

Reply:
Thanks Hanna - so technically it's '... add a `gov-uk` prefix to any selectors that find components using the `data-module` attribute.'?

Reply:
Sounds good to me 👍

Reply:
Added in https://github.com/alphagov/govuk-frontend/pull/1492/commits/217e12a762789d8040c56dc7831cebc0c9282801